steward/src/commands/cosmos_to_eth.rs

```diff
@@ -10,12 +10,13 @@ use std::{process::exit, time::Duration};
 
 const TIMEOUT: Duration = Duration::from_secs(60);
 
-/// Send Cosmos to the Eth chain
+/// Send Cosmos to the Eth chain. args is a Vector of Strings and flag_no_batch is boolean
 #[derive(Command, Debug, Default, Parser)]
 pub struct CosmosToEthCmd {
+    /// <gravity_denom> <amount> <cosmos_key> <eth_dest> <times>
     #[clap(short, long)]
     pub args: Vec<String>,
-
+    /// boolean
```

Comment:
Knowing the expected type is good, but it should also explain what it means to set `flag_no_batch`.